sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/InnerExceptionCountExceededException.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.Serialization;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// This exception is used to notify the user that the set of inner exceptions has been trimmed because it exceeded our allowed send limit.
+    /// </summary>
+    [Serializable]
```

Comment:
do we need this?

Reply:
Will get this removed, at this point there is no need for this attribute.